src/Components/Home.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react'
+import './Welcome.css'
```

Comment:
Don't use global CSS classes. Use something like PostCSS Modules to import the CSS classes and apply them directly on your elements. Example:
```
import classes from './Welcome.css'

<button className={classes.logoutButton} />
```